### PR TITLE
Add Yekan(B-Yekan) font

### PR DESCRIPTION
**Pull request by SajadRahimi1**
add another Persian font that name is Yekan.

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -10,12 +10,11 @@ class MyApp extends StatelessWidget {
...
-      home: MyHomePage(),
-      theme: ThemeData(
-        textTheme: PersianFonts.vazirTextTheme,
-      )
-    );
+        title: 'Persian Fonts Example',
+        home: MyHomePage(),
+        theme: ThemeData(
+          textTheme: PersianFonts.vazirTextTheme,
+        ));
```
Comment (mahdyar) [inline example/lib/main.dart]: Please don't change indention and leave it as it is, it probably happened because you use some code formatter.

**Review by mahdyar — CHANGES_REQUESTED** [review]
Hello @SajadRahimi1 and thank you for your contribution;
please review the suggestion(s) and then ask for another review. :)

**Review by SajadRahimi1 — COMMENTED** [review]
change example/lib/main.dart like main code

**Review by mahdyar — APPROVED** [review]
Thank you, @SajadRahimi1!

**Pull request merged**